oktapam/resource_server_checkout_settings_test.go

```diff
@@ -0,0 +1,186 @@
+package oktapam
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/atko-pam/pam-sdk-go/client/pam"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/okta/terraform-provider-oktapam/oktapam/constants/attributes"
+)
+
+const testAccServerCheckoutSettingsCreateConfigFormat = `
+provider "oktapam" {}
```

Comment:
Create base config string with group, resource group and project. Then, reuse that for creating and updating settings.